rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction.adoc

```diff
@@ -0,0 +1,107 @@
+# Usecase: Extract Data from a Legacy System
+
+Extracting data from legacy systems is a common use case for RPA. 
+
+This example illustrates the following tasks:
+
+* Copy the contents from a PDF file to a text file.
+* Compare strings.
+* Copy a substring from a text file.
+* Find text on a screen using optical character recognition (OCR).
+* Interact with a screen using mouse actions and key strokes.
+* Use activity parameters to transfer data and to determine which path to follow in the model.
+
+## Example Scenario
+
+Imagine you work as an accountant at Mythical University. Mythical University has ordered a travel package for one of their field excursions. You have received an email and saved the attached order confirmation to your computer.
+
+This is what an order confirmation of your travel agency Wonderlust Travel typically looks like:
+
+[[confirmationletter-image]]
+image::usecase-1-wonderlusttravels-orderconfirmation-pdf.png["Letter containing a logo, headers, text and tables. The order ID is contained in a table in a header in the middle of the page. The label says 'Order ID'.", 75%]
+
+You need to search the order ID in your order tracking system to get the corresponding tracking number, which you need to reference in your payment.
+
+Unfortunately your order tracking system is very old and accountants can only access it by typing in and reading from a terminal screen:
+
+[[trackingsystem-allentries-image]]
+image::as400ordertrackingscreen.png["Legacy order tracking system showing green glowing text in a terminal. Above a list of orders, there is an entry field for the order id labeled 'Order-ID' and a button labeled 'Search' next to it.", 75%]
+
+You click the search field, enter the order ID, and click 'Search'.
```

Comment:
Should Search be bold?

Is the field named "Order-ID"?

Reply:
I didn't bold the ui elememts of the example application to differentiate between the example and our product. In this example application the field is indeed named 'Order-ID'

Reply:
I see now - you're not referring to the field in this case - so ignore that comment.

On line 30, Search seems to be tagged italic.

Reply:
This is strange. In this preview it looks like italics but in the code it's not...